report/out.go

```diff
@@ -57,3 +56,76 @@ func Out(f string, infos []map[string]string) error {
 
 	return nil
 }
+
+func makeTitleStyle(fh *excelize.File) int {
+	template := `{
+		"fill": {
+			"type": "pattern",
+			"color": [
+				"#D3D3D3"
+			],
+			"pattern": 1
+		},
+		"border": [
+			{
+				"type": "bottom",
+				"color": "000000",
+				"style": 1
+			},
+			{
+				"type": "top",
+				"color": "000000",
+				"style": 1
+			},
+			{
+				"type": "left",
+				"color": "000000",
+				"style": 1
+			},
+			{
+				"type": "right",
+				"color": "000000",
+				"style": 1
+			}
+		]
+	}`
+
+	id, err := fh.NewStyle(template)
+	if err != nil {
+		log.Fatalln("failed to cell style setting.")
+	}
+	return id
+}
+
+func makeBodyStyle(fh *excelize.File) int {
```

Comment:
The setting of the result part is also made into a separate function as a workaround.